ecpy/tasks/tasks/util/log_task.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015 by Ecpy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Logging Task.
+
+"""
+from atom.api import (Unicode, set_default)
+import logging
+
+from ...base_tasks import SimpleTask
+
+class LogTask(SimpleTask):
+    """ Task logging a message. Loopable.
+
+    """
+
+    #: Message to log when the task is executed.
+    message = Unicode().tag(pref=True, feval=True)
```

Comment:
Here we should use fmt not feval as I think this will be more natural to the user.